inference_segmentation.py

```diff
@@ -390,6 +394,9 @@ def main(params: Union[DictConfig, dict]) -> None:
     device = set_device(gpu_devices_dict=gpu_devices_dict)
 
     clahe_clip_limit = get_key_def('clahe_clip_limit', params['tiling'], expected_type=Number, default=0)
+    heatmap_dtype = get_key_def('heatmap_dtype', params['inference'], default=np.uint16)
```

Comment:
@valhassan dtype will default to uint16 to capture more details from original logits (originally as float32 or 16)